### DISCUSSION TITLE
PDF Export of Vue Component

### DISCUSSION THREAD

**Discussion by mecjos** [quasarframework/quasar#9508] [General - Components / Directives / etc]
Hi. I need a pdf export plugin or package for quasar. I want to create some templates for pdf as a seperate Vue component then export them as pdf. For example an order list in qtable or as list. Is there any plugin or package to do that? Thank you.

**Comment by ontwikkelfabriek** [comment]
>  export them as pdf.

With export you mean : the user of the Quasar App can download pdf's?

If so you can use the export file utils of Quasar:
https://quasar.dev/quasar-utils/other-utils#export-file

**Reply by mecjos** [reply]
No, converting html into pdf and download like laravel dompdf. I don't want to do it at serverside. There is a package (jspdf) but it's not useful.

**Reply by ontwikkelfabriek** [reply]
@mecjos 

Did you try:
https://github.com/kempsteven/vue-html2pdf

**Reply by mecjos** [reply]
Thank you.. I think this will be helpful.. I will try..

**Reply by ontwikkelfabriek** [reply]
@mecjos 

If you would post about your experience/findings with that vue-html2-pdf library (or other solutions to convert html/vue to pdf) and Quasar it would be much appreciated!
